### DISCUSSION TITLE
Open PR blocks other upgrade PRs from opening

### DISCUSSION THREAD

**Discussion by privatenumber** [renovatebot/renovate#13309] [Request Help] [answered]
## Problem
I'm using Renovate in https://github.com/privatenumber/minification-benchmarks to auto open PRs for [these minifiers](https://github.com/privatenumber/minification-benchmarks/blob/239ad6f0e62f7593ed30c52f493ce7e47d9432e5/renovate.json#L18-L23).

When a Renovate dependency upgrade PR is open for one dependency, it seems to prevent other dependency upgrade PRs from opening. I have `prConcurrentLimit20` enabled, but there is currently only 1 PR.

For example, currently https://github.com/privatenumber/minification-benchmarks/pull/70 is open, but `@swc/core` is also outdated (1.2.123 -> 1.2.124) and there won't be a PR for it until I merge this one.

## Question
How can I configure Renovate so that multiple upgrade PRs can be open at the same time (one dependency upgrade per PR)?

**Comment by rarkins** [comment]
Do you have any logs to support your assumptions?

**Comment by viceice** [comment, marked as answer]
You've edited the branch, so renovate stops to not overwrite your changes
![Screenshot_20211230-085838.png](https://user-images.githubusercontent.com/1798109/147732714-498fe26d-97f7-43b8-b152-5468c72a0767.png)

**Reply by privatenumber** [reply]
Ahh I see. Good catch. Is there a way to configure Renovate to ignore that and restart off of the base branch?

I have a GitHub Action that runs the benchmarks and commits it on every PR against `master`.

**Reply by rarkins** [reply]
You can tick the rebase/retry checkbox in any PR.

You can potentially also use this in your config to avoid the problem: https://docs.renovatebot.com/configuration-options/#gitignoredauthors

**Reply by privatenumber** [reply]
Perfect! I just confirmed that resolves my issue.

Thank you so much @viceice & @rarkins
